docs/guides/references/migration-guide.mdx

```diff
@@ -6,7 +6,66 @@ title: Migration Guide
 
 This guide details the changes and how to change your code to migrate to Cypress
 version 13.0.
-[See the full changelog for version 13.0](/guides/references/changelog#13-0-0).
+[See the full changelog for version 13.0](https://docs.cypress.io/guides/overview/why-cypress).
+
+### Cypress Cloud Test Replay
+
+[Test Replay](https://docs.cypress.io/guides/overview/why-cypress) is enabled by default in `v13.0` of the Cypress App.
+
+You may need to whitelist `capture.cypress.io` if you work with a strict VPN. See our FAQ section about [VPN subdomain whitelisting](/faq/questions/cloud-faq#Im-working-with-a-restrictive-VPN-Which-subdomains-do-I-have-to-allow-on-my-VPN-for-Cypress-Cloud-to-work-properly).
+
+You can [opt out](https://docs.cypress.io/guides/overview/why-cypress) of this feature in Cloud project-level settings.
+
+<!---
+todo: replace urls for merge to v13 branch:
+
+changelog- /guides/references/changelog#13-0-0
+test replay- /guides/cloud/debugging/test-replay
+opt out- /guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay
+--->
+
+### Video updates
+
+#### `video` is set to false by default
+
+Whether Cypress will capture a video of the tests run with `cypress run`.
+
+You can continue recording video, by passing `video: true`, if you want video locally or you want video for some other reason, like in non-Chromium browsers where [Test Replay](https://docs.cypress.io/guides/overview/why-cypress) is not available.
+
+:::cypress-config-example
+
+```ts
+{
+  video: true
+}
+```
+
+:::
+
+#### `videoUploadOnPasses` configuration option has been removed
+
+Whether Cypress will process, compress, and upload videos to Cypress Cloud even when all tests in a spec file are passing.
+
+See our [Control which videos to keep and upload to Cypress Cloud](/guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Cypress-Cloud) guide with code examples to discard captured video of passing tests.
```

Comment:
```suggestion
Most users used `videoUploadOnPasses` as a way to skip the time to compress and upload videos to the Cloud. Since we're turning off `videoCompression` by default, this configuration option does not offer the time saving value that it once would. 

If you want to prevent a passing test from uploading to the Cloud, we recommend deleting the video using our [guide with code examples to discard captured video of passing tests](/guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Cypress-Cloud).
```